Adds notes for 4.11.5 release (#50540)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

[id="ocp-4-11-5"]
=== RHSA-2022:6536 - {product-title} 4.11.5 bug fix and security update

Issued: 2022-09-20

{product-title} release 4.11.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:6536[RHSA-2022:6536] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6535[RHBA-2022:6535] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.11.5 --pullspecs
----

[id="ocp-4-11-5-known-issues"]
==== Known issues

* Sharding the default Ingress Controller breaks the {product-title} factory routes such as `canary`, `oauth`, and `console`. As a workaround, you can manually add matching labels and expressions to the routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024946[*BZ#2024946*])

[id="ocp-4-11-5-bug-fixes"]
==== Bug fixes

> * Previously, after a `routeSelector` update, the Ingress Controller cleared the route status before the router deployment finished rolling out, resulting in the status being repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])

Reviewer: ```suggestion
* Previously, an update of the `routeSelector` cleared the route status of the Ingress Controller before the router deployment finished rolling out, which resulted in the status being repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])
```

Reviewer 2: ```suggestion
*  Previously, update of the `routeSelector` cleared the route status of the Ingress Controller prior to the router deployment. As a result, the route status repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])
```
Just an additional idea, not a blocker. I sometimes get lost in bug fixes, due to their compactness and chain of events. As a result, I try to make each sentence as short as I can.

Author: Thanks for the suggestions! I have updated.
